### PR TITLE
Feature/#29 light dark toggle

### LINKED ISSUE #29: Dark Mode / Light Mode
As a user I would be able to switch between light/dark mode to choose which contrast helps me focus on the content.

### PR DESCRIPTION
**Pull request by dpajek**
---
name: Light Dark Toggle
about: 
- Split theme into light/dark themes and added identifiers for colors that change between themes (e.g., backgroundMenu, backgroundPrimary, textPrimary, etc)
- Added toggle to navbar and mobile menu
- Updated styles so pages match the light theme designs
title: ''
labels: ''
assignees: dpajek
---

Resolves #29

### CODE REVIEW

File: client/components/layout/styles.js [hunk 1]
```diff
@@ -67,14 +67,53 @@ export const Navbar = styled.nav`
...
+
+export const ToggleSwitchSlider = styled.span`
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: ${props => props.theme.colors.backgroundPrimary};
+  border-radius: 26px;
```
Comment (ZumDeWald) [inline client/components/layout/styles.js:95]: The toggle looks great, except when in dark mode you can't see the container the toggle switch is in so it looks just like a green circle sitting in the nav 😅.

Maybe something like this on line 96 to be able to see the switch background:
`${props => props.isDarkTheme && 'filter: contrast(1.3)'};`
I tried it on my local and it makes the switch BG a little darker and the the toggle looks more like a toggle.

**Review by ZumDeWald — CHANGES_REQUESTED** [review]
@dpajek I saw the Sun and Moon files in `public`, but didn't see you bring them in anywhere. I think the toggle looks fine without them so if they don't get used please remember to remove them.

I like the changes overall and they work smoothly. I had one suggestion for the toggle switch in dark mode, but that's it from me!

File: client/components/layout/styles.js (hunk 1 above)
Comment (dpajek) [reply client/components/layout/styles.js:95]: @ZumDeWald Thanks for your feedback! Whoops, I forgot to remove those files. (I was originally implemented something with sun/moon images, but I changed it to the toggle switch).

Oohh, that's a nice approach to create a constrasting background without having to create additional color variables. I'll add it to the PR. Thanks :)

**Review by ZumDeWald — APPROVED** [review]
Looks good and works smoothly for me.

**Review by cMikolai — APPROVED** [review]
Works very neat, thank you!

**Pull request merged**